### DISCUSSION TITLE
Multi Series Barchart Tooltip

### DISCUSSION THREAD

**Discussion by akinayturan** [reaviz/reaviz#35] [Q&A]
<img width="749" alt="Screen Shot 2021-01-23 at 15 18 59" src="https://user-images.githubusercontent.com/3206344/105578067-a16dd100-5d8e-11eb-9a27-8b37d387156e.png">

As seen in the picture, the tooltip areas appear in different places each time, is there a solution for this?

`<BarChart
                    width={550}
                    height={350}
                    data={datam}
                    series={
                        <BarSeries
                            layout="vertical"
                            type="grouped"
                            colorScheme={[
                                '#16b723',
                                '#2b7477',
                            ]}
                            tooltip={<TooltipArea
                                followCursor={true}
                                modifiers={{
                                    offset: '5px, 5px'
                                }}
                                color={'#CCC'}
                            />}
                        />
                    }
                />`

**Comment by amcdnl** [comment]
You need to add these CSS variables to fix the tooltip background: https://reaviz.io/?path=/story/docs-getting-started-quick-start--page#configuration.

The position is setup to be screen aware so that it will fit the position so that it doesn't go off screen.

**Reply by akinayturan** [reply]
<img width="477" alt="Screen Shot 2021-01-25 at 16 35 27" src="https://user-images.githubusercontent.com/3206344/105712799-605ef380-5f2b-11eb-95fa-4ff4c6df24b5.png">


<img width="757" alt="Screen Shot 2021-01-25 at 16 37 52" src="https://user-images.githubusercontent.com/3206344/105713026-b03dba80-5f2b-11eb-95d4-c43b2182acd0.png">


the color is not the problem position is the main problem ^^

**Reply by akinayturan** [reply]
<img width="1075" alt="Screen Shot 2021-01-25 at 16 37 44" src="https://user-images.githubusercontent.com/3206344/105745151-07a15200-5f4f-11eb-8aae-d7368aa3d499.png">

<img width="845" alt="Screen Shot 2021-01-25 at 20 50 03" src="https://user-images.githubusercontent.com/3206344/105745170-0cfe9c80-5f4f-11eb-902c-ea03f99d8cbf.png">

it changing every time :)
